fe/fe-core/src/test/java/com/starrocks/analysis/SelectStmtTest.java

```diff
@@ -343,4 +343,27 @@ public void testScalarCorrelatedSubquery() {
             Assert.fail("Should not throw an exception");
         }
     }
+
+    @Test
+    public void testMultiDistinctMultiColumnWithLimit() throws Exception {
+        String[] sqlList = {
+                "select count(distinct k1, k2), count(distinct k3) from db1.tbl1 limit 1",
+                "select * from (select count(distinct k1, k2), count(distinct k3) from db1.tbl1) t1 limit 1",
```

Comment:
how about the sql : select count(distinct k1, k2), count(distinct k3) from db1.tbl1 group by k4 limit 1?

Reply:
group-by test cases added